### DISCUSSION TITLE
Unable apply for student develop pack

### DISCUSSION THREAD

**Discussion by Yi-Baoxiang** [community/community#52529] [GitHub Education]
I want to complain! I have been refusing my application for a student package:

       Previously, I couldn't capture the materials clearly using the device, but I refused because I couldn't upload them on my phone due to low credibility,
       Finally, I used my campus card as a material to take photos and submit them, but I have consistently rejected me several times for this reason:You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.
       I changed my name to my real name. What exactly is written in Bio? Is my name in Chinese or English? What should I do.

我要投诉！我申请学生包一直给我拒绝：
之前用设备拍摄材料又拍不清楚给我拒绝了，用手机上传又因为可信度不高给我拒绝，最后我用校园卡作为材料拍照提交又一直以这样的理由拒绝我好几次：
You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.
我明明姓名都改为了真名，Bio里面到底写什么啊，姓名到底用中文还是英文啊，怎么办啊

**Comment by samus-aran** [comment]
Hi @Yi-Baoxiang 👋 

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Let us know how you get on 👍

**Comment by whitewatercn** [comment]
老哥别急，可以参考我的这个帖子再逝一逝 https://xlog.whitewater.wang/Github_Student_Developer_Pack
Be patient, you can refer to this post to try again https://xlog.whitewater.wang/Github_Student_Developer_Pack
